paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java

```diff
@@ -108,11 +108,25 @@ public List<SnapshotVO> getSnapshot(MetadataDTO dto) {
                     internalRow -> {
                         SnapshotVO build =
                                 SnapshotVO.builder()
-                                        .setSnapshotId(internalRow.getLong(0))
-                                        .setSnapshotId(internalRow.getLong(1))
-                                        .setCommitIdentifier(internalRow.getLong(3))
-                                        .setCommitTime(
+                                        .snapshotId(internalRow.getLong(0))
+                                        .schemaId(internalRow.getLong(1))
+                                        .commitUser(internalRow.getString(2).toString())
+                                        .commitIdentifier(internalRow.getLong(3))
+                                        .commitKind(internalRow.getString(4).toString())
+                                        .commitTime(
                                                 internalRow.getTimestamp(5, 3).toLocalDateTime())
+                                        .baseManifestList(internalRow.getString(6).toString())
+                                        .deltaManifestList(internalRow.getString(7).toString())
+                                        .changelogManifestList(
+                                                internalRow.getString(8) != null
+                                                        ? internalRow.getString(8).toString()
+                                                        : "")
```

Comment:
Why not null?

Reply:
During the test, it was found that internalRow.getString(8) may return null, and if no judgment is made, a null pointer will be reported.